spec/models/template_spec.rb

```diff
@@ -53,7 +53,7 @@
       character1 = create(:character, template: template)
       character2 = create(:character, template: template, template_name: "nickname")
       character3 = create(:character, template: template, screenname: "screen_name")
-      expect(template.plucked_characters).to eq([[character1.id, character1.name], [character2.id, "#{character2.name} | #{character2.template_name}"], [character3.id, "#{character3.name} | #{character3.screenname}"]])
+      expect(template.plucked_characters).to match_array([[character1.id, character1.name], [character2.id, "#{character2.name} | #{character2.template_name}"], [character3.id, "#{character3.name} | #{character3.screenname}"]])
```

Comment:
Is plucked_characters not a consistent order?

Reply:
I seem to have lost this change in the rebase and don't have any new failures there, so I... guess... it is? idk why I thought it wasn't.